migrations.go

```diff
@@ -206,11 +245,25 @@ func RegisteredMigrations() []Migration {
 // - version - prints current db version.
 // - set_version - sets db version without running migrations.
 func Run(db DB, a ...string) (oldVersion, newVersion int64, err error) {
-	return RunMigrations(db, RegisteredMigrations(), a...)
+	return DefaultGroup.Run(db, a...)
 }
 
 // RunMigrations is like Run, but accepts list of migrations.
 func RunMigrations(db DB, migrations []Migration, a ...string) (oldVersion, newVersion int64, err error) {
+	g := &Group{
```

Comment:
It could be good to deprecate this method and instead provide a mechanism for setting the list of migrations on a Group.

Reply:
Sounds good, but let's deprecate it in a separate PR>